chaoscenter/upgrade-agents/control-plane/versions/upgrade-manager.go

```diff
@@ -158,55 +214,129 @@ func (m *UpgradeManager) getUpgradePath() map[string]UpgradeExecutor {
 			NextVersion:    "",
 			VersionManager: nil,
 		},
+
+		"3.6.0": {
+			NextVersion:    "3.7.0",
+			VersionManager: nil,
+		},
+		"3.7.7-beta8": {
```

Comment:
All these old versions where no upgrades are available can be removed from this map